src/transports/http.js

```diff
@@ -7,11 +7,17 @@
 
 const http       = require("http");
 const https      = require("https");
-const path       = require("path");
-const express    = require("express");
 const bodyParser = require("body-parser");
+const cors       = require("cors");
 
-const utils = require(path.resolve(__dirname, "../utils"));
+const utils = require("../utils");
+
+// For some reason you can't browserify express.
+// We don't need it anyway, for browser support.
+let express;
```

Comment:
Maybe instead of this, do something like: https://github.com/substack/node-browserify#browser-field